frontend/apps/marketing/src/components/carousels/actionBlockCarousel/ActionBlockCarousel.tsx

```diff
@@ -103,6 +106,9 @@ const ActionBlockCarousel: React.FC<ActionBlockCarouselProps> = ({
                   : undefined
               }
               background={background}
+              tag={
```

Comment:
Can we toss in a quick unit test for this?

Reply:
We don't have unit tests for the carousels bc of Swiper, and this is already being tested on the standalone Action Block component that's used in the carousel. https://github.com/code-dot-org/code-dot-org/blob/5eca948749901eef15bec2deb72b4cd763106609/frontend/apps/marketing/src/components/actionBlocks/defaultActionBlock/__tests__/ActionBlock.test.tsx#L82-L116